internals/daemon/api.go

```diff
@@ -101,7 +101,7 @@ var API = []*Command{{
 }, {
 	Path:        "/v1/notices",
 	ReadAccess:  UserAccess{},
-	WriteAccess: UserAccess{},
+	WriteAccess: AdminAccess{},
```

Comment:
This will make posting user notices impossible. Is that the intent? I thought we already had fairly good checks inside the notices code?

Reply:
Good spotting! This was actually not working earlier (when Pebble notices UID support was introduced), and fixed (unintentionally) by the CheckAccess PR. Reverted to `UserAccess{}` and commented on why this is the case.